src/lsp/capabilities/server.rs

```diff
@@ -256,6 +256,8 @@ pub fn collect_dynamic_registrations(
         ));
     }
 
+    registrations.push(create_registration("custom/ViewSyntaxTree", ()));
```

Comment:
```suggestion
    registrations.push(create_registration("cairo/ViewSyntaxTree", ()));
```

don't these need to be namespaced?

Reply:
Done independently of ur comment, but it does not